stress/src/main.rs

```diff
@@ -47,6 +49,214 @@ const DEFAULT_WRITE_REGION_COUNT: u64 = 5;
 const DEFAULT_WRITE_SYNC: bool = true;
 const DEFAULT_REUSE_DATA: bool = false;
 
+#[derive(Debug, Clone, Parser)]
+#[clap(
+    name = "Engine Stress (stress)",
+    about = "A stress test tool for Raft Engine",
+    author = crate_authors!(),
+    version = crate_version!(),
+    setting = AppSettings::DontCollapseArgsInUsage,
+)]
+struct ControlOpt {
+    /// Path of raft-engine storage directory
+    #[clap(
+        long = "path",
+        takes_value = true,
+        about = "Set the data path for Raft Engine"
+    )]
+    path: String,
+
+    #[clap(
+        long = "time",
+        takes_value = true,
+        default_value = formatcp!("{}", DEFAULT_TIME.as_secs()),
+        value_name = "time[s]",
+        about = "Set the stress test time"
+    )]
+    time: String,
```

Comment:
maybe u64? so we don't need to manually parse the string.

Reply:
To avoid parsing manually, I change type of most parameters, except `write_sync` and `reuse_data` because it will trigger 'default_value is meaningless for bool' error